packages/shared/src/modules/section-statuses/libs/validation-schemas/section-status-create-body.validation-schema.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from "zod";
+
+import { SectionStatus } from "../enums/enums.js";
+
+const sectionStatusCreateBody = z
+	.object({
+		courseSectionId: z.coerce.number(),
```

Comment:
```suggestion
		courseSectionId: z.number(),
```
If it's for body we don't need coerce